Students/Pontinteu Crinela/Assignment3/src/javasmmr/zoowsome/models/animals/Butterfly.java

```diff
@@ -0,0 +1,12 @@
+package javasmmr.zoowsome.models.animals;
+
+public class Butterfly extends Insect {
+	
+	public Butterfly() {
```

Comment:
You also have to provide a constructor with parameters so you can manually set the attributes of a Butterfly. This new constructor will be used for twist2.

The same for the rest of your classes that are not abstract.